OpenSim/Simulation/Model/Appearance.h

```diff
@@ -186,7 +152,6 @@ class OSIMSIMULATION_API Appearance : public Object {
         constructProperty_opacity(1.0);
         // White by default, shows as a shade of gray
         constructProperty_color(SimTK::Vec3(1.0)); 
```

Comment:
Sure, thanks @tkuchida though they end up in XML as three numbers but that's more readable. :+1: